### PR TITLE
BUGZ-1156: Allow collapsing of all multi spheres

### PR DESCRIPTION
**Pull request by luiscuenca**
https://highfidelity.atlassian.net/browse/BUGZ-1156

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20846/HighFidelity-Beta-PR16059-1570c59.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20846/HighFidelity-Beta-Interface-PR16059-1570c59.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20846/HighFidelity-Beta-PR16059-1570c59.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20846/HighFidelity-Quest-Beta-PR16059-1570c59.apk)

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20856/HighFidelity-Beta-PR16059-02e730a.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20856/HighFidelity-Quest-Beta-PR16059-02e730a.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20856/HighFidelity-Beta-PR16059-02e730a.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20856/HighFidelity-Beta-Interface-PR16059-02e730a.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20856/HighFidelity-Beta-PR16059-02e730a.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20856/HighFidelity-Beta-Interface-PR16059-02e730a.dmg

File: libraries/physics/src/MultiSphereShape.cpp [hunk 1]
```diff
@@ -317,15 +318,28 @@ void MultiSphereShape::spheresFromAxes(const std::vector<glm::vec3>& points, con
...
-            SphereShapeData newSphere;
-            newSphere._position = 0.5f * (spheres[0]._position + spheres[1]._position);
-            newSphere._radius = spheres[maxRadiusIndex]._radius;
-            spheres.clear();
-            spheres.push_back(newSphere);
+    if (sphereCount > 1) {
+        bool collapsed = false;
+        for (size_t i = 0; i < spheres.size() - 1; i++) {
+            for (size_t j = i + 1; j < spheres.size(); j++) {
+                if (i != j) {
```
Comment (hyperlogic) [inline libraries/physics/src/MultiSphereShape.cpp]: nit: if j starts at i + 1, i can never equal j. So you don't need this if.

**Pull request merged**